src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -15,105 +15,110 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <configurator>
-  <domain name="CHIP"/>
-
-  <enum name="AccessControlEntryPrivilegeEnum" type="enum8">
-    <cluster code="0x001F"/>
-    <item name="View" value="0x01"/>
-    <item name="Proxy View" value="0x02"/>
-    <item name="Operate" value="0x03"/>
-    <item name="Manage" value="0x04"/>
-    <item name="Administer" value="0x05"/>
-  </enum>
-
-  <enum name="AccessControlEntryAuthModeEnum" type="enum8">
-    <cluster code="0x001F"/>
-    <item name="PASE" value="0x01"/>
-    <item name="CASE" value="0x02"/>
-    <item name="Group" value="0x03"/>
-  </enum>
-
-  <enum name="ChangeTypeEnum" type="enum8">
-    <cluster code="0x001F"/>
-    <item name="Changed" value="0x00"/>
-    <item name="Added" value="0x01"/>
-    <item name="Removed" value="0x02"/>
-  </enum>
-
-  <struct name="AccessControlTargetStruct">
-    <cluster code="0x001F"/>
-    <item fieldId="0" name="Cluster" type="cluster_id" isNullable="true"/>
-    <item fieldId="1" name="Endpoint" type="endpoint_no" isNullable="true"/>
-    <item fieldId="2" name="DeviceType" type="devtype_id" isNullable="true"/>
-  </struct>
-
-  <struct name="AccessControlEntryStruct" isFabricScoped="true">
-    <cluster code="0x001F"/>
-    <item fieldId="1" name="Privilege" type="AccessControlEntryPrivilegeEnum" isFabricSensitive="true"/>
-    <item fieldId="2" name="AuthMode" type="AccessControlEntryAuthModeEnum" isFabricSensitive="true"/>
-    <item fieldId="3" name="Subjects" type="int64u" isNullable="true" array="true" isFabricSensitive="true"/>
-    <item fieldId="4" name="Targets" type="AccessControlTargetStruct" isNullable="true" array="true" isFabricSensitive="true"/>
-  </struct>
-
-  <struct name="AccessControlExtensionStruct" isFabricScoped="true">
-    <cluster code="0x001F"/>
-    <item fieldId="1" name="Data" type="octet_string" length="128" isFabricSensitive="true"/>
-  </struct>
-
-  <cluster>
-    <domain>General</domain>
-    <name>Access Control</name>
-    <code>0x001F</code>
-    <define>ACCESS_CONTROL_CLUSTER</define>
-    <client init="false" tick="false">true</client>
-    <server init="false" tick="false">true</server>
-    <description>The Access Control Cluster exposes a data model view of a
+	<domain name="CHIP" />
+
+	<enum name="AccessControlEntryPrivilegeEnum" type="enum8">
+		<cluster code="0x001F" />
```

Comment:
So this is using 8-space indent?

Is that configurable?  It will make these files very hard to read due to some of the lines ending up pretty long....